pages/common/odps-ddl.md

```diff
@@ -0,0 +1,23 @@
+# odps ddl
+
+> Run ddl task in ODPS (Open Data Processing Service).
```

Comment:
Hm, I wonder if we should also expand "ddl". Also, shouldn't it be "tasks" (plural)?